cg/models/flow_cell/flow_cell.py

```diff
@@ -101,10 +103,19 @@ def get_sample_sheet_path_hk(self) -> Path:
         return self._sample_sheet_path_hk
 
     def get_flow_cell_run_dir(self) -> Path:
-        """Return the flow cells run directory if the FlowCellsDirectoryData was initialised with Demultiplexed runs dir."""
+        """
+        Return the flow cells run directory
+        if the FlowCellsDirectoryData was initialised with Demultiplexed runs dir.
```

Comment:
I think this will work either way
```suggestion
        Return the flow cell run directory.
```

Reply:
Yes I just want to make this explicit and indicate that this class has two different use cases (which it likely should not)

Reply:
I would then change it for 
```suggestion
        Return the flow cells run directory regardless of the path used to initialise the FlowCellsDirectoryData.
```